pom.xml

```diff
@@ -234,6 +234,13 @@
 
 	<dependencies>
 		<dependency>
+			<groupId>org.atmosphere</groupId>
+			<artifactId>atmosphere-runtime</artifactId>
+			<version>2.4.12</version>
+			<scope>provided</scope>
```

Comment:
I think `<scope>compile</scope>` with `<optional>true</optional>` is correct since this is an optional feature. I will send a follow up PR.